web/server/README.md

```diff
@@ -98,21 +98,21 @@ openssl speed rsa2048 rsa4096
 
 ### Select TLS version
 
-Since version 1.21 Netdata accepts `tls key` and `tls certificate` as replacement of the options `ssl key` and `ssl certificate`, 
- but you do not need to change your `netdata.conf` case you are using old `ssl` variables, because old options are kept due compatibility.
+Beginning with version 1.21, Netdata accepts `tls key` and `tls certificate` as replacement of the settings `ssl key` and `ssl certificate`. 
 
-Netdata also brings two new options related to TLS, now you can specify the TLS version and the ciphers that you want to use:
+You do not need to change your `netdata.conf` if you are using the previous `ssl` settings, because these are kept for backward compatibility.
+
+You can also specify the TLS version and the ciphers that you want to use:
 
 ```conf
 [web]
     tls version = 1.3
     tls ciphers = TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256
 ```
 
-case you do not specify these options, Netdata will use the highest options available on your system.
+If you do not specify these options, Netdata will use the highest options available on your system.
```

Comment:
For at least the ciphers, this isn't true. Netdata will use the default cipher list based on the selected protocol, which may not be the most secure possible.

Reply:
Case we use the highest protocol version, 1.3 right now, we will discard a lot of ciphers that are not considered safe.

Reply:
That doesn't mean we automatically use the most secure ciphers though, which is what the statement above implies.. Take a look at the standard list for 1.2 for example, almost half of them are considered to at best be weak and at worst completely broken.

I'd suggest the following alternative:
```suggestion
If you do not specify these options, Netdata will use the highest available protocol version on your system and the default cipher list for that protocol provided by your TLS implementation.
```

Reply:
I accepted the suggestion, I am finishing the tests here, and I will commit everything in few minutes. Thank you!

Reply:
Done!